### PR TITLE
improve UI, add appID for 'Pages' & update manifest

### PR DESCRIPTION
**Pull request by AnanyaBarthakur1**

### CODE REVIEW

**Review by MarcinJuraszek — APPROVED** [review]

**Review by aeunms — APPROVED** [review]
I do see some of the same ' ' vs '' stuff as from the other PR, but I don't think it's a huge deal. And it's already being fixed in the other PR I think.

File: apps/sample-app/src/components/PeopleAvatars.tsx [hunk 1]
```diff
@@ -59,48 +69,55 @@ export const PeopleAvatarList: React.FC<AvatarProps> = (props: AvatarProps) => {
...
+        <div>
+          {AvatarItemList.map(a => (
+            <Tooltip
+              content={
+                <>
+                  <Text weight="semibold" as="span">
+                    {a.name}
+                  </Text>
+                  <MenuList>
+                    {capabilities.map(capability => (
```
Comment (MarcinJuraszek) [inline apps/sample-app/src/components/PeopleAvatars.tsx:85]: I think it's OK to leave it as is, but I'm curious why you think this is better than having the `isSupported` check here, something like:

```
{mail.isSupported() && <MenuItem onClick={() => handleMail(a)}>Mail</MenuItem>
```

File: apps/sample-app/src/components/PeopleAvatars.tsx [hunk 2]
```diff
@@ -59,48 +69,55 @@ export const PeopleAvatarList: React.FC<AvatarProps> = (props: AvatarProps) => {
...
-            </Text>
-          </Tooltip>
-        ))}
-      </div>
+      <>
+        <Title3 block className="paddingClass">
+          People to Meet Today
+        </Title3>
+        <div>
+          {AvatarItemList.map(a => (
```
Comment (MarcinJuraszek) [inline apps/sample-app/src/components/PeopleAvatars.tsx:77]: nit: use `avatar` instead of `a`, to make the loop body easier to read.

**Review by MarcinJuraszek — DISMISSED** [review]

File: apps/sample-app/src/components/PeopleAvatars.tsx (hunk 1 above)
Comment (AnanyaBarthakur1) [reply apps/sample-app/src/components/PeopleAvatars.tsx:85]: I agree that this is more readable and simpler to understand. I'll try incorporating it this way and see if the UI remains the same (which seems like it should!) and push it in a future PR?

Comment (MarcinJuraszek) [reply apps/sample-app/src/components/PeopleAvatars.tsx:85]: sounds good!

**Review by MarcinJuraszek — DISMISSED** [review]

**Review by MarcinJuraszek — APPROVED** [review]

**Review by aeunms — APPROVED** [review]

**Pull request merged**